README.md

```diff
@@ -129,15 +165,16 @@ public class MainApplication extends Application implements ReactApplication {
     MobileCore.setApplication(this);
     MobileCore.setLogLevel(LoggingMode.DEBUG);
     try {
-          Lifecycle.registerExtension();
           Signal.registerExtension();
-          com.adobe.marketing.mobile.edge.identity.Identity.registerExtension();
+          Lifecycle.registerExtension();
           Edge.registerExtension();
+          com.adobe.marketing.mobile.edge.identity.Identity.registerExtension();
           Consent.registerExtension();
-          //Messaging.registerExtension();
-          //Assurance.registerExtension();
-          //UserProfile.registerExtension();
-          //com.adobe.marketing.mobile.Identity.registerExtension();
+          //Messaging.registerExtension(); //enable this extension if you installed @adobe/react-native-aepmessaging
+          //UserProfile.registerExtension(); //enable this extension for store user profile attributes
+          //Assurance.registerExtension(); //enable this extension for inspecting and validating the app
+          //com.adobe.marketing.mobile.Identity.registerExtension(); //enable for Adobe solution Identity extension 
+
       MobileCore.configureWithAppID("yourAppID");
       MobileCore.start(new AdobeCallback() {
         @Override
```

Comment:
let's remove the lifecycle call in the two examples below

Reply:
@emdobrin @yangyansong-adbe 
I added Lifecycle back and with a Note section.  Please see if it looks ok.
I also added Edge Lifecycle gitbook link to the Core lifecycle readme.

Reply:
Looks good to me.

Reply:
I added description to each extensions, does it look too busy now?

Reply:
Looks much better now, thanks Calise!